commands/import.go

```diff
@@ -0,0 +1,251 @@
+	RunE:    withClient(importListIncompleteCmdF),
+}
+
+var ImportListJobsCmd = &cobra.Command{
+	Use:     "jobs [importJobID]",
+	Example: " import list jobs",
+	Short:   "List import jobs",
+	Args:    cobra.MaximumNArgs(1),
+	RunE:    withClient(importListJobsCmdF),
+}
+
+var ImportProcessCmd = &cobra.Command{
+	Use:     "process [importname]",
+	Example: " import process 35uy6cwrqfnhdx3genrhqqznxc_import.zip",
+	Short:   "Start an import job",
+	Args:    cobra.ExactArgs(1),
+	RunE:    withClient(importProcessCmdF),
+}
+
+func init() {
+	ImportUploadCmd.Flags().Bool("resume", false, "Set to true to resume an incomplete import upload.")
+	ImportUploadCmd.Flags().String("upload", "", "The ID of the import upload to resume.")
+	ImportListJobsCmd.Flags().Int("limit", 10, "The maximum number of jobs to show.")
+	ImportListCmd.AddCommand(
+		ImportListAvailableCmd,
+		ImportListIncompleteCmd,
+		ImportListJobsCmd,
+	)
+	ImportCmd.AddCommand(
+		ImportUploadCmd,
+		ImportListCmd,
+		ImportProcessCmd,
+	)
+	RootCmd.AddCommand(ImportCmd)
+}
+
+func importListIncompleteCmdF(c client.Client, command *cobra.Command, args []string) error {
+	isLocal, _ := command.Flags().GetBool("local")
+	userID := "me"
+	if isLocal {
+		userID = fakeUserID
+	}
+
+	uploads, resp := c.GetUploadsForUser(userID)
+	if resp.Error != nil {
+		return fmt.Errorf("failed to get uploads: %w", resp.Error)
+	}
+
+	var imports []*model.UploadSession
+	for i := range uploads {
+		if uploads[i].Type == model.UploadTypeImport {
+			imports = append(imports, uploads[i])
+		}
+	}
+
+	if len(imports) == 0 {
+		printer.Print("No incomplete import uploads found")
+		return nil
+	}
+
+	for _, us := range imports {
+		completedPct := float64(us.FileOffset) / float64(us.FileSize) * 100
+		printer.PrintT(fmt.Sprintf("  ID: {{.Id}}\n  Name: {{.Filename}}\n  Uploaded: {{.FileOffset}}/{{.FileSize}} (%0.0f%%)\n", completedPct), us)
+	}
+
+	return nil
+}
+
+func importListAvailableCmdF(c client.Client, command *cobra.Command, args []string) error {
+	imports, resp := c.ListImports()
+	if resp.Error != nil {
+		return fmt.Errorf("failed to list imports: %w", resp.Error)
+	}
+
+	if len(imports) == 0 {
+		printer.Print("No import files found")
+		return nil
+	}
+
+	for _, name := range imports {
+		printer.Print(name)
+	}
+
+	return nil
+}
+
+func importUploadCmdF(c client.Client, command *cobra.Command, args []string) error {
+	filepath := args[0]
+
+	file, err := os.Open(filepath)
+	if err != nil {
+		return fmt.Errorf("failed to open import file: %w", err)
+	}
+	defer file.Close()
+
+	info, err := file.Stat()
+	if err != nil {
+		return fmt.Errorf("failed to stat import file: %w", err)
+	}
+
+	shouldResume, _ := command.Flags().GetBool("resume")
+	var us *model.UploadSession
+	var resp *model.Response
+	if shouldResume {
+		uploadID, err := command.Flags().GetString("upload")
+		if err != nil || !model.IsValidId(uploadID) {
+			return errors.New("required upload ID is missing or invalid")
```

Comment:
nit: This error will implicitly tell that the upload ID is required 😅 
```suggestion
			return errors.New("upload ID is missing or invalid")
```